### ISSUE TITLE
Change output fieldname "trail_ids" to "segment_ids" in trailheads.geojson

### ISSUE THREAD

**Issue by ryanbranciforte** [codeforamerica/OpenTrails#81]
Jmoe, is this what you were referring to?
![screen shot 2014-08-13 at 5 05 44 pm](https://cloud.githubusercontent.com/assets/2602563/3914472/435502f6-2347-11e4-8d16-4725b7b7a65e.png)

**Comment by jmoe** [comment]
Yes. Currently the converter seems to be outputting trail_ids instead of trail_segment_ids. But maybe there is rationale for this? @alanjosephwilliams would know.

**Comment by alanjosephwilliams** [comment]
I will look into our transformer functions and see if I can identify what needs to be changed.

It looks like the key name needs to change, but that the value is fine.
